.github/workflows/hpu-gaudi2.yml

```diff
@@ -26,6 +26,58 @@ jobs:
         - 80
       options: --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice
 
+    env:
+      PT_HPU_LAZY_MODE: 0
+      TEST_LIST: |
```

Comment:
Tagging @mrwyattii for his thoughts on running the test list this way vs trying to skip tests that are unsupported.

Reply:
@loadams @mrwyattii ,

Any update on the PR. Should we merge this?

Reply:
Hi @vshekhawat-hlab - we are working on fixing the other CI tests then we will get this merged.

Reply:
These should be resolved now, thanks!